console/utils/oauth/gitee_api.py

```diff
@@ -257,7 +257,8 @@ def get_repo_detail(self, full_name, *args, **kwargs):
         access_token, _ = self._get_access_token()
         repo_list = []
         repos = [self.api.get_repo(full_name)]
-        for repo in repos:
+        for rp in repos:
```

Comment:
did api change?

Reply:
gitee api return `data` and `total`，Instead of just returning data